test/test-generate-kmp.ts

```diff
@@ -13,17 +13,37 @@ const sources = [
     ["SEN", 4816],
   ]),
 ];
-const modelId = "nrc.en.mtnt";
+const modelID = "nrc.en.mtnt";
 
 test("it should generate a kmp file", async (t) => {
-  const kmp = await generateKmp(langName, bcp47Tag, sources, modelId);
+  const kmp = await generateKmp(langName, bcp47Tag, sources, modelID);
   t.assert(kmp.byteLength > 0);
 
   const new_zip = new JSZip();
   const zip = await new_zip.loadAsync(kmp);
   const isKmpExist = zip.file("kmp.json");
-  const isModelExist = zip.file("nrc.en.mtnt.model.js");
+  const isModelExist = zip.file(modelID + ".model.js");
 
   t.assert(isKmpExist != null);
   t.assert(isModelExist != null);
 });
+
+const modelID2 = "national_research_council_canada.str.sencoten";
+
+test("The modelID should determines the filename of the model.js file in kmp archive", async (t) => {
+  const kmp = await generateKmp(langName, bcp47Tag, sources, modelID2);
+  const new_zip = new JSZip();
+  const zip = await new_zip.loadAsync(kmp);
```

Comment:
I know this was present in the last test, but why is `new_zip` in `snake_case` and not in `camelCase` like all other variable names?